d2b_migrate.module

```diff
@@ -89,12 +112,24 @@ function template_preprocess_d2b_migrate_page(&$variables) {
 function d2b_migrate_task_list($active = NULL) {
   // Default list of tasks.
   $tasks = array(
-    'start' => t('Start'),
-    'source' => t('Source database'),
-    'analysis' => t('Database analysis'),
-    'backup' => t('Database backup'),
-    'files' => t('Backup files'),
-    'restore' => t('Migrate database'),
+    'start' => t('<a href="@url">Start</a>', array(
+      '@url' => url('d2b-migrate/start'),
+    )),
+    'source' => t('<a href="@url">Source database</a>', array(
+      '@url' => url('d2b-migrate/source'),
+    )),
+    'analysis' => t('<a href="@url">Database analysis</a>', array(
+      '@url' => url('d2b-migrate/analysis'),
+    )),
+    'backup' => t('<a href="@url">Database backup</a>', array(
+      '@url' => url('d2b-migrate/backup'),
+    )),
+    'files' => t('<a href="@url">Backup files</a>', array(
+      '@url' => url('d2b-migrate/files'),
+    )),
+    'restore' => t('<a href="@url">Migrate database</a>', array(
+      '@url' => url('d2b-migrate/restore'),
+    )),
```

Comment:
Perhaps consider using `l()` instead of `url()` here? See:
- https://docs.backdropcms.org/api/backdrop/1/search/url
- https://docs.backdropcms.org/api/backdrop/1/search/l